app/middleware/locationValidator.js

```diff
@@ -2,6 +2,13 @@ const log = require('../lib/logger');
 const renderer = require('../middleware/renderer');
 const locationValidator = require('../lib/locationValidator');
 
+function setLocationLabel(res, location) {
+  if (location) {
+    // eslint-disable-next-line no-param-reassign
+    res.locals.locationLabel = 'Enter a valid postcode';
```

Comment:
Are we moving towards all text being defined in middleware? Just trying to understand why we have the page title hard coded in the view but location label is defined in locals. In principle, it's a good idea but not sure of the benefit it brings us as I think it is currently only used in a single place.